File: 백준/Bronze/2480. 주사위 세개/주사위 세개.c
#include <stdio.h>

int main(void) {
    int A = 0;
    int B = 0;
    int C = 0;
    int result = 0;
    
    scanf("%d %d %d", &A, &B, &C);
    
    if(A == B && B == C)
        result = 10000 + A * 1000;
    else if(A == B || A == C)
        result = 1000 + A * 100;
    else if(B == C)
        result = 1000 + B * 100;
    else
        result = ((A > B ? A : B) > C ? (A > B ? A : B) : C) * 100;
    printf("%d\n", result);
    return 0;
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CsTutorial {
    class Program {
        static void Main(string[] args) {
            string[] str = Console.ReadLine().Split();
            int H = int.Parse(str[0]);
            int M = int.Parse(str[1]);
            int C = int.Parse(str[2]);
            int result = 0;
            
            if(H == M && M == C)
                result = 10000 + H * 1000;
            else if(H == M || H == C)
                result = 1000 + H * 100;
            else if(M == C)
                result = 1000 + M * 100;
            else
                result = ((H > M ? H : M) > C ? (H > M ? H : M) : C) * 100;
            
            Console.WriteLine(result);
        }
    }
}